### PR TITLE
New package: plume-creator-0.66.1

### PR DESCRIPTION
**Pull request by VargMon**

### CODE REVIEW

**Comment by VargMon** [comment]
Tested and working on i686 and x86_64 Void Linux glibc.

**Comment by VargMon** [comment]
Cross compiling also works with xbps-src ;) in my tests.

File: srcpkgs/plume-creator/template [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+makedepends="
+ qt5-devel qt5-declarative-devel qt5-tools-devel qt5-quickcontrols2-devel qt5-multimedia-devel qt5-svg-devel hunspell-devel
+ libzip-devel quazip-devel quazip-qt5-devel"
+short_desc="An open-source tool for novelists"
+maintainer="VargMon <VargMon98@gmail.com>"
+license="GPL-3"
+homepage="http://plume-creator.eu/"
+configure_args="plume-creator-all.pro"
+distfiles="
+ ${UBUNTU_SITE}/universe/p/plume-creator/${pkgname}_${_debianversion}.orig.tar.xz
```
[the remark this answers is not in the document]
Comment (VargMon) [reply srcpkgs/plume-creator/template]: True, but it's either that or I'd have to in-house the patches... and well I'd thought I might have to do something that later on when the upstream project's latest version comes out, unless hopefully I woudn't have to. However, without the debian patches it won't compile.

File: srcpkgs/plume-creator/template [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	hostmakedepends+=" 
+ qt5-multimedia-devel qt5-devel qt5-declarative-devel qt5-quickcontrols2-devel hunspell-devel
+ libzip-devel quazip-devel quazip-qt5-devel"
+fi
+
+post_extract() {
+	mv -f ${wrksrc}/../debian ${wrksrc}/
+	mv -f ${wrksrc}/../resources/themes ${wrksrc}/
+	mv -f ${wrksrc}/../src/pics/* ${wrksrc}/src/pics/
+	patch -p1 <${wrksrc}/debian/patches/git.2014-01-09.patch
```
[the remark this answers is not in the document]
Comment (VargMon) [reply srcpkgs/plume-creator/template]: I didn't see any patch vars in the documentation. :(

[the remark this answers is not in the document]
Comment (VargMon) [reply srcpkgs/plume-creator/template]: Okay making mentioned changes

**Comment by Vaelatern** [comment]
#12076
also it's now merged.

**Comment by VargMon** [comment]
@Vaelatern Thanks. :) I've now figured out xbps-src's way of patching and am working on getting the_integrated_ patching system working but for some unknown reason on this particular package I can't seem to get it to patch correctly. Also feel free to email me if there's any issues.

**Pull request closed without merging**